ezyrb/ann.py

```diff
@@ -0,0 +1,83 @@
+"""
+Module for Artificial Neural Network (ANN) Prediction.
+"""
+import torch
+import torch.nn as nn
+import numpy as np
+from .approximation import Approximation
+
+class ANN(Approximation):
+    """
+    Feed-Forward Artifical Neural Network (ANN).
+    : param int trained_epoch: number of already trained iterations.
+    : param criterion: Loss definition (Mean Squared). 
```

Comment:
You can merge these two lines with `:param torch.nn.modules.loss.MSELoss criterion`

Reply:
Thanks for the suggestion!